tests/test_cli_pinentry.py

```diff
@@ -0,0 +1,25 @@
+import time
+
+__author__ = "Rick Elrod"
+__copyright__ = "(c) 2022 Red Hat, Inc."
+__license__ = "MIT"
+
+
+def test_pinentry_simple(tmux_session, gpg_home_with_secret_key, unsigned_project_with_checksum_manifest):
+    """Test that we can sign a file with a pinentry program."""
+    home = gpg_home_with_secret_key
+    window = tmux_session.new_window(window_name="test_pinentry_simple")
```

Comment:
This is so cool, hadn't seen it before.

Reply:
It's used in ansible-navigator tests. Originally I wrote some automation to use tmux commands directly and do something similar, then someone found libtmux that wraps it and makes it into a nice Python API.